src/ring_theory/algebra.lean

```diff
@@ -258,6 +258,11 @@ namespace algebra
 variables (R : Type u) (S : Type v) (A : Type w)
 variables [comm_ring R] [comm_ring S] [ring A] [algebra R S] [algebra S A]
 include R S A
+
+/-- `comap R S A` is a type alias for `A`, and has an R-algebra structure defined on it
+  when `algebra R S` and `algebra S A`. -/
+/- This is done to avoid a type class search with meta-variables `algebra R ?m_1` and 
+  `algebra ?m_1 A -/
 def comap : Type w := A
```

Comment:
```suggestion
@[nolint] def comap : Type w := A
```
Otherwise `comap` will fail the unused arguments test.

Reply:
There's actually no reason for it to have all the typeclass arguments.

Reply:
You're right. But even when we omit those, we still want it to take `R` and `S` as arguments, right? (If I'm understanding the doc string correctly.) So it will still need `@[nolint]` for those.

Reply:
Done